Project.toml

```diff
@@ -0,0 +1,21 @@
+name = "Clustering"
```

Comment:
I think the REQUIRE is only used by the `attobot` and older versions of Julia. Not for 1.0 and later.

Reply:
If the package has *REQUIRE* file, it will be used by Julia 1.0 and 1.1.
AFAIK, the plan is to automatically generate .toml files for all community-maintained Julia packages, once .toml would be fully supported by the package infrastructure (e.g. *attobot*).